### DISCUSSION TITLE
History of project

### DISCUSSION THREAD

**Discussion by yringler** [simolus3/drift#1547] [General]
I'm in the middle of moving my app to Drift, and have been thrilled with the ergonomic and delightful API (for context, I'm coming from C# - entity framework, ADO, dapper etc).
It seems to me to strike a perfect compromise between ease of quality implementation and ease of use.
Is there another project / API which inspired this design? Or is wholly original (at least, as much as anything can be considered original).

**Comment by simolus3** [comment]
Thank you for the kind words, I'm glad you're enjoying Drift :)

The overall structure of the Dart API was inspired by [Exposed](https://github.com/JetBrains/Exposed) for Kotlin. Drift files and the static analyzer for SQL have taken inspiration from [SQLDelight](https://github.com/cashapp/sqldelight), but I tried to put my own focus on bridging SQL and the Dart APIs.
I think I can claim that lots smaller features making up Drift (like using zones for transactions, remote/isolate support, Dart components in SQL queries) are specific to this project. Or at least I like to believe that Drift's community and I came up with them :D
